web-registry/src/pages/Buyers/Buyers.EcologicalCreditCardsSection.tsx

```diff
@@ -24,6 +26,17 @@ const BuyersEcologicalCreditCardsSection: React.FC<
           linkComponent={Link}
           sx={{ mb: { xs: 5, sm: 7.5 } }}
           {...card}
+          children={
```

Comment:
It's best to pass CardMedia as children instead of props:
`<Card><CardMedia /></Card>`

Reply:
interesting! didn't know about this, thanks for the heads up, it'll probably take a few more times before i understand it. I just pushed a commit making the change, not sure if it's right but it seems to be working